content/cpp/concepts/maps/terms/swap/swap.md

```diff
@@ -0,0 +1,127 @@
+---
+Title: '.swap()'
+Description: 'Exchanges the content of two maps.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Objects'
+  - 'OOP'
+  - 'Classes'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`.swap()`** function exchanges the content of two maps of the same type.
+
+## Syntax
+
+The `map1` is the map whose contents are exchanged with the contents of `map2`.
+
+```pseudo
+map1.swap(map2);
+```
+
+> **Note:** If `map1` and `map2` are not of the same type then a compilation error will occur.
```

Comment:
```suggestion
> **Note:** If `map1` and `map2` are not of the same type (i.e., they do not have the same key and value types), a compilation error will occur because `std::map::swap()` requires both maps to have the same type.
```